### ISSUE TITLE
authRequestHeaders not recognized as a configuration option

### ISSUE THREAD

**Issue by daften** [traefik/traefik#7445]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I updated Traefik to 2.3.2 and used the new authRequestHeaders option for forwardAuth added recently.
I updated my forwardAuth config to authelia to look like this:

```
[http.middlewares]
  [http.middlewares.middlewares-authelia]
    [http.middlewares.middlewares-authelia.forwardAuth]
      address = "http://authelia:9091/api/verify?rd=https://authelia.example.com"
      trustForwardHeader = true
      authRequestHeaders = "X-Fowarded-Proto,X-Forwarded-Host"
      authResponseHeaders = ["Remote-User", "Remote-Groups"]
```

### What did you see instead?

The logs on startup show that the configuration is not recognized: 
```
time="2020-10-21T03:27:23Z" level=error msg="Cannot start the provider *file.Provider: /rules/middlewares.toml: field not found, node: authRequestHeaders"
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.3.2
Codename:     picodon
Go version:   go1.15.3
Built:        2020-10-19T18:36:22Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Spread out over multiple files, but the most relevant is shown above

```toml
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

There is only one relevant line, which is shown above. If more info is needed, I can give it, but I prefer to keep data secure :)

```
(paste your output here)
```

**Comment by daften** [comment]
For those running into this, I fixed it on authelia's side with the options added here, which is probably a better solution (for me at least). For me cookie was the biggest header so I could strip what I wanted, chances of going over the max size were big.

https://github.com/authelia/authelia/pull/944/files

**Comment by ldez** [comment]
Hello,

`authRequestHeaders` options will be available in the next version (v2.4).

- https://github.com/traefik/traefik/pull/7226
- https://github.com/traefik/traefik/blob/v2.3/CHANGELOG.md

Also the syntax will be (in the v2.4):

```toml
[http.middlewares]
  [http.middlewares.test-auth.forwardAuth]
    address = "https://example.com/auth"
    authResponseHeaders = ["X-Auth-User", "X-Secret"]
```

https://doc.traefik.io/traefik/master/middlewares/forwardauth/#authresponseheaders

Thanks anyways.
